process/process.md

```diff
@@ -0,0 +1,291 @@
+# Preface
+
+This document defines processes adopted by Xen-Troops project. It
```

Comment:
Only "introducing code changes" processes. Other processes we have not yet developed (i.e. task breakdown/estimation, triage, ...)